test/byContract/IexecPoco/04_finalize.test.ts

```diff
@@ -356,6 +356,135 @@ describe('IexecPoco2#finalize', async () => {
         expect(task.resultsCallback).to.equal('0x'); // deal without callback
     });
 
+    [
+        {
+            testInfo: 'Get 10% of the kitty when these 10% are greater than MIN_KITTY', // (MIN_KITTY=1RLC)
+            workerpoolPriceToFillKitty: 33_333_333_367,
+            expectedKittyBeforeFinalize: 10_000_000_010, // ~10 RLC
+            expectedSchedulerKittyPartReward: 1_000_000_001, // ~1.01 RLC (expectedKittyBeforeFinalize x 10%)
+        },
+        {
+            testInfo: 'Get at least MIN_KITTY if available',
+            workerpoolPriceToFillKitty: 3_333_333_334,
+            expectedKittyBeforeFinalize: 1_000_000_000, // 1 RLC
+            expectedSchedulerKittyPartReward: 1_000_000_000,
+        },
+        {
+            testInfo: 'Get all kitty when the kitty is lower than MIN_KITTY',
+            workerpoolPriceToFillKitty: 3_333_333_330,
+            expectedKittyBeforeFinalize: 999_999_999, // ~0.99 RLC
+            expectedSchedulerKittyPartReward: 999_999_999,
+        },
+    ].forEach((testArgs) => {
+        const {
+            testInfo,
+            workerpoolPriceToFillKitty,
+            expectedKittyBeforeFinalize,
+            expectedSchedulerKittyPartReward,
+        } = testArgs;
+        it(`Should finalize task with scheduler kitty part reward (${testInfo})`, async () => {
+            // Fill kitty
+            const kittyAddress = await iexecPoco.kitty_address();
+            const kittyFillingDealVolume = 1;
+            const kittyFillingSchedulerDealStake = await iexecWrapper.computeSchedulerDealStake(
+                workerpoolPriceToFillKitty,
+                kittyFillingDealVolume,
+            );
+            const kittyFillingSchedulerTaskStake =
+                kittyFillingSchedulerDealStake / kittyFillingDealVolume;
+            const kittyFilling = await iexecWrapper.signAndMatchOrders(
```

Comment:
I don't think `kittyFilling` "alone" is the right name, like the others that have kittyFillingX with X as the description of what is in the variable

Reply:
At least here , I like it after with `kittyFilling.dealId` etc